src/frontend/src/utils/routes.ts

```diff
@@ -38,7 +38,7 @@ const CR_BY_ID = `/cr/:id`; // short version of CHANGE_REQUESTS_BY_ID
 const CHANGE_REQUESTS_NEW = CHANGE_REQUESTS + `/new`;
 const CHANGE_REQUESTS_NEW_WITH_WBS = CHANGE_REQUESTS_NEW + `?wbsNum=`;
 const CHANGE_REQUESTS_OVERVIEW = CHANGE_REQUESTS + `/overview`;
-
+const CHANGE_REQUESTS_NEW_WITH_WBS_WORKPACKAGE = CHANGE_REQUESTS_NEW + `?wbsNum=`;
```

Comment:
CHANGE_REQUESTS_NEW_WITH_WBS_WORKPACKAGE and CHANGE_REQUESTS_NEW_WITH_WBS are duplicates, go ahead and get rid of one of them